### PR TITLE
Feat#66595 CRUD contact

### PR DESCRIPTION
**Pull request by Bui-Cong-Tri**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/66595)

## WHAT this PR does?
- Tạo UI, validation, controller cho contact
- Sửa lại logic và giao diện của user

## Notes Impacted Areas:
- N/A.

## Attachments:
- User Show:
image
- Contact create:
image
- Contact show:
image
- Contact edit:
image

### CODE REVIEW

**Comment by Bui-Cong-Tri** [comment]
Ready

File: app/Http/Controllers/UserController.php [hunk 1]
```diff
@@ -45,6 +46,7 @@ public function store(CreateUserRequest $request)
 
     public function show(User $user)
     {
+        $user->load('contacts');
         return view('users.show')->with('user', $user);
```
Comment (ghost) [inline app/Http/Controllers/UserController.php]:
```suggestion
        $user->load('contacts');

        return view('users.show')->with('user', $user);
```

File: resources/views/contacts/create.blade.php [hunk 2]
```diff
@@ -0,0 +1,21 @@
+<x-app-layout>
+    <x-slot name="header">Add new contact</x-slot>
+    <div class="py-8">
+        <div class="mx-auto max-w-7xl sm:px-6 lg:px-8 bg-white shadow-sm sm:rounded-lg p-6">
+            <form action="{{ route('contacts.store') }}" method="post">
+                @csrf
+                <input type="hidden" value="{{ Auth::id() }}" name="user_id" />
+                <x-label for="name">Name:</x-label>
```
Comment (ghost) [inline resources/views/contacts/create.blade.php]: localization

**Comment by Bui-Cong-Tri** [comment]
Ready

**Comment by Bui-Cong-Tri** [comment]
Ready

**Review by ghost — APPROVED** [review]

**Pull request merged**